### PR TITLE
D2K - Add Harkonnen 9b

### LINKED ISSUE #14392: Get the remaining Harkonnen missions into the playtest
Since @MustaphaTR has all of them locally finished and ready for review, we should try to get them all reviewed and merged before the playtest branch-off (or before the 2nd playtest at the latest), as the first completed campaign would make for a nice advertiser.

- [x] Harkonnen 5 #14388
- [x] Harkonnen 7 #14426
- [x] Harkonnen 8 #14427
- [x] Harkonnen 9a #14429
- [ ] Harkonnen 9b #14434

### PR DESCRIPTION
**Pull request by MustaphaTR**
~~Depends on #14433. Player starts with no MCV/ConYard in this mission and should build one. You need Heavy Factory Upgrade to build an MCV.~~

I didn't make seperate Smuggler vs Atreides and Smuggler vs Corrino sides, because that would require 8 different smuggler players.

Also fixes a small issue on Har6a and b.

Closes #14392.

### CODE REVIEW

**Comment by pchote** [comment]
#14433 has been merged.

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by reaperrr** [comment]
Needs rebase.

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by ltem** [comment]
Needs a rebase.

**Comment by MustaphaTR** [comment]
Rebased.

File: mods/d2k/maps/harkonnen-09b/harkonnen09b.lua [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+BuildFremen = function()
+	if APalace.IsDead or APalace.Owner ~= atreides_main then
+		return
+	end
+
+	APalace.Produce("fremen")
+	APalace.Produce("fremen")
+
+	Trigger.AfterDelay(DateTime.Seconds(1), function()
```
Comment (ltem) [inline mods/d2k/maps/harkonnen-09b/harkonnen09b.lua]: A delay of 1s is too short, the Fremen won't be "idle" and therefore won't be counted. Increasing it to 2s solved it for me. I don't know on what that behavior depends, so maybe it should be increased further...

Comment (ltem) [reply mods/d2k/maps/harkonnen-09b/harkonnen09b.lua]: `Harkonnen 9a` would also benefit from this change.

**Comment by ltem** [comment]
Except the comment above I didn't find any issues so 👍 from me after fixing it.

File: mods/d2k/maps/harkonnen-09b/harkonnen09b.lua (hunk 1 above)
Comment (MustaphaTR) [reply mods/d2k/maps/harkonnen-09b/harkonnen09b.lua]: I think it would be better if we don't take much risk and make it 5 sec.

**Comment by MustaphaTR** [comment]
Updated. Also removed some stuff related to #14459 from 9a and 9b.

**Review by ltem — DISMISSED** [review]

**Comment by ltem** [comment]
No issues found after the small update. Needs a rebase. Afterwards 👍 from me.

**Comment by reaperrr** [comment]
Same here 👍

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
I've decided to rebase this myself so we could get it merged now.

[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/806a68b8683e794833f0d44e887e6e7d660fec6b...429671c8f73657b56443ad21513c6c0b88fc82c7)

**Pull request merged**